tools/install-scripts/install.ps1

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env pwsh
+# Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
+# TODO(everyone): Keep this script simple and easily auditable.
+# Forked from Deno's install.ps1 script
+
+$ErrorActionPreference = 'Stop'
+
+if ($v) {
+  $Version = "v${v}"
+}
+if ($Args.Length -eq 1) {
+  $Version = $Args.Get(0)
+}
+
+$TypstInstall = $env:TYPST_INSTALL
+$BinDir = if ($TypstInstall) {
+  "${TypstInstall}\bin"
+} else {
+  "${Home}\.typst\bin"
+}
+
+$TypstZip = "$BinDir\typst.zip"
+$TypstExe = "$BinDir\typst.exe"
+$Target = 'x86_64-pc-windows-msvc'
+
+$DownloadUrl = if (!$Version) {
+  "https://github.com/typst/typst/releases/latest/download/typst-${Target}.zip"
+} else {
+  "https://github.com/typst/typst/releases/download/${Version}/typst-${Target}.zip"
+}
+
+if (!(Test-Path $BinDir)) {
+  New-Item $BinDir -ItemType Directory | Out-Null
+}
+
+curl.exe -Lo $TypstZip $DownloadUrl
+
+tar.exe xf $TypstZip -C $BinDir --strip-components=1
```

Comment:
Do windows computers have `curl` and `tar` by default these days?

Reply:
https://devblogs.microsoft.com/commandline/tar-and-curl-come-to-windows/